blocks/overline-block/features/overline/default.jsx

```diff
@@ -1,11 +1,84 @@
 import React from "react";
+import PropTypes from "prop-types";
+import { useFusionContext } from "fusion:context";
+import { useEditableContent } from "fusion:content";
+import getProperties from "fusion:properties";
+import getTranslatedPhrases from "fusion:intl";
+import { Overline as OverlineOutput, formatURL } from "@wpmedia/arc-themes-components";
 
-import { Overline as OverlineOutput } from "@wpmedia/shared-styles";
+const BLOCK_CLASS_NAME = "b-overline";
 
-const Overline = () => <OverlineOutput />;
+const Overline = (props) => {
+	const { globalContent, arcSite } = useFusionContext();
+	const { editableContent } = useEditableContent();
+	const phrases = getTranslatedPhrases(getProperties(arcSite).locale || "en");
+
+	const { customText, customUrl, editable, story, className = "" } = props;
+
+	const sourceContent = story || globalContent || {};
```

Comment:
This block relies on `globalContent` only

Reply:
I believe I narrowed it down to global content logic only